projects/packages/videopress/src/client/admin/components/video-grid/index.tsx

```diff
@@ -30,26 +30,28 @@ const getThumbnail = ( { video }: { video: VideoPressVideo } ): React.ReactNode
  * @param {VideoGridProps} props - Component props.
  * @returns {React.ReactNode} - VideoGrid react component.
  */
-const VideoGrid = ( { videos, count = 6, onVideoDetailsClick }: VideoGridProps ) => {
+const VideoGrid = ( { videos, count = 6, onVideoDetailsClick, loading }: VideoGridProps ) => {
 	const gridVideos = videos.slice( 0, count );
 
 	const handleClickWithIndex = ( index, callback ) => () => {
 		callback?.( videos[ index ] );
 	};
+
 	return (
 		<div className={ styles.wrapper }>
 			<Container fluid horizontalSpacing={ 0 } horizontalGap={ 0 }>
 				{ gridVideos.map( ( video, index ) => {
 					return (
 						<Col key={ index } sm={ 4 } md={ 4 } lg={ 4 }>
-							<ConnectVideoCard
+							<VideoCard
```

Comment:
This change should be at least discussed. There is an intentionality in creating the Connect component.

Reply:
The idea behind this component is to delete the task to pull the data independently of the context where the card renders.

It only requires the video ID. In this way, the component can handle global states from its PoV (like when the app is loading all videos which this case, as well as individual states such as when the video is being edited, added, or removed).

Why are you removing this implementation?

Reply:
@retrofox I’m not removing it. It’s only the default export now.